src/ovirtapi/index.js

```diff
@@ -203,7 +207,7 @@ const OvirtApi = {
       contentType: 'application/xml',
     })
   },
-  addNewSnapshot ({ vmId, snapshot }: { vmId: string, snapshot: Object }): Promise<Object> {
+  addNewSnapshot ({ vmId, snapshot }: { vmId: string, snapshot: ApiSnapshotType }): Promise<Object> {
```

Comment:
I think you need `SnapshotType` here since 2 lines down there is a `OvirtApi.internalSnapshotToOvirt({ snapshot })`